e2e/rbd_helper.go

```diff
@@ -252,9 +256,9 @@ func validateEncryptedPVCAndAppBinding(pvcPath, appPath, kms string, f *framewor
 		return fmt.Errorf("%v not equal to crypt", mountType)
 	}
 
-	if kms == "vault" {
+	if kmsIsVault(kms) || kms == "vaulttokens" {
```

Comment:
helper function for vaulttokens? similar to `kmsIsVault` `kmsIsVaultToken`?

Reply:
Okay got it may be when we add more options we can do it. Should not block this PR from getting merged